samples/langchain_quick_start.ipynb

```diff
@@ -20,14 +22,1094 @@
+    "\n",
+    "\n",
+    "SpannerVectorStore.init_vector_store_table(\n",
+    "    instance_id=instance_id,\n",
+    "    database_id=database_id,\n",
+    "    table_name=sample_vector_table_name,\n",
+    "    metadata_columns=[\n",
+    "        TableColumn(\"show_id\", \"STRING(MAX)\"),\n",
+    "        TableColumn(\"type\", \"STRING(MAX)\"),\n",
+    "        TableColumn(\"country\", \"STRING(MAX)\"),\n",
+    "        TableColumn(\"date_added\", \"STRING(MAX)\"),\n",
+    "        TableColumn(\"release_year\", \"INT64\"),\n",
+    "        TableColumn(\"rating\", \"STRING(MAX)\"),\n",
+    "        TableColumn(\"duration\", \"STRING(MAX)\"),\n",
+    "        TableColumn(\"listed_in\", \"STRING(MAX)\"),\n",
+    "    ],\n",
+    "    primary_key=\"langchain_id\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "KG6rwEuJLNIo"
+   },
+   "source": [
+    "### Try inserting the documents into the vector table\n",
+    "\n",
+    "Now we will create a vector_store object backed by our vector table in the Spanner database. Let's load the data from the documents to the vector table. Note that for each row, the embedding service will be called to compute the embeddings to store in the vector table. Pricing details can be found [here](https://cloud.google.com/vertex-ai/pricing)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {
+    "id": "Wo4-7EYCIFF9"
+   },
+   "outputs": [],
+   "source": [
+    "from langchain_google_vertexai.embeddings import VertexAIEmbeddings\n",
+    "\n",
+    "# Initialize the embedding service. In this case we are using version 003 of Vertex AI's textembedding-gecko model. In general, it is good practice to specify the model version used.\n",
+    "embeddings_service = VertexAIEmbeddings(\n",
+    "    model_name=\"textembedding-gecko@003\", project=project_id\n",
+    ")\n",
+    "\n",
+    "\n",
+    "vector_store = SpannerVectorStore(\n",
+    "    embedding_service=embeddings_service,\n",
+    "    instance_id=instance_id,\n",
+    "    database_id=database_id,\n",
+    "    table_name=sample_vector_table_name,\n",
+    "    metadata_columns=[\n",
+    "        \"show_id\",\n",
+    "        \"type\",\n",
+    "        \"rating\",\n",
+    "        \"country\",\n",
+    "        \"date_added\",\n",
+    "        \"release_year\",\n",
+    "        \"duration\",\n",
+    "        \"listed_in\",\n",
+    "    ],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "fr1rP6KQ-8ag"
+   },
+   "source": [
+    "Now let's try to put the documents data into the vector table. Here is a code example to load the first 5 documents in the list."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {
+    "id": "CTks8Cy--93B"
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['83118fcc-6c38-4b93-9dab-03270c9cf2a9',\n",
```

Comment:
I think that the output is still showing in this file, could you clear the cells first